cmd/boot-script-service/api_test.go

```diff
@@ -1,6 +1,6 @@
 // MIT License
 //
-// (C) Copyright [2021] Hewlett Packard Enterprise Development LP
+// (C) Copyright [2021-2022] Hewlett Packard Enterprise Development LP
```

Comment:
are you changing copyright years on files that you didnt modify?
If the file has no change, it doesnt need the copyright change.   Is the license checker failing that?

Reply:
Yes, I changed the copyright on the files that I didn't change. Those were the files that the license checker was failing on.

Reply:
talk to Mikhail about this, bc we arent going to just bulk update years b/c of the license checker.